src/ig/IGExporter.ts

```diff
@@ -513,61 +626,86 @@ export class IGExporter {
               continue;
             }
 
-            const resource: ImplementationGuideDefinitionResource = {
+            const referenceKey = `${resourceJSON.resourceType}/${resourceJSON.id}`;
+            const newResource: ImplementationGuideDefinitionResource = {
               reference: {
-                reference: `${resourceJSON.resourceType}/${resourceJSON.id}`
-              },
-              name: resourceJSON.id, // will be overwritten w/ title or name where applicable
-              description: resourceJSON.description
+                reference: referenceKey
+              }
             };
+            const configResource = (this.config.resources ?? []).find(
+              resource => resource.reference?.reference == referenceKey
+            );
 
-            if (pathEnd === 'examples') {
-              const exampleUrl = resourceJSON.meta?.profile?.find(
-                url =>
-                  this.pkg.fish(url, Type.Profile) ?? this.fhirDefs.fishForFHIR(url, Type.Profile)
+            if (!(configResource?.omit === true)) {
+              const existingIndex = this.ig.definition.resource.findIndex(
+                r => r.reference.reference === referenceKey
               );
-              if (exampleUrl) {
-                resource.exampleCanonical = exampleUrl;
-              } else {
-                resource.exampleBoolean = true;
+              const existingResource =
+                existingIndex >= 0 ? this.ig.definition.resource[existingIndex] : null;
```

Comment:
I was a little confused when first reviewing this.  I think I understand now that this code about `existingResource` is to support the use case where someone wants to override a FSH-generated resource with a user-provided resource (which might be helpful to get around a known bug in the FSH generation).  Maybe put a comment somewhere indicating the use case we're trying to handle?